### ISSUE TITLE
Very large utterances make detail viewer unusuable

### ISSUE THREAD

**Issue by MichaelGoodale** [MontrealCorpusTools/ISCAN#210]
If you have a very large utterance(say 600 seconds), the detail viewer is unusuably slow. It might be good to allow for someone to just look at part of an utterance at a time. Not sure exactly how this would work, but perhaps people could supply a set of times they want to look.

**Comment by msonderegger** [comment]
@MichaelGoodale I assume this is low priority because if "utterances" defined correctly there shouldn't be any which are anywhere near this long?

**Comment by MichaelGoodale** [comment]
@msonderegger Yes this only really would happen if the pauses aren't encoded correctly, or if a speaker is just really not taking any breaths.
